src/velero/values/values.yaml

```diff
@@ -0,0 +1,56 @@
+kubectl:
+  # Container Level Security Context for the 'kubectl' container of the crd jobs. Optional.
+  # See: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container
+  containerSecurityContext:
+    capabilities:
+      drop:
+        - ALL
+  # Annotations to set for the upgrade/cleanup job. Optional.
+  annotations:
+    sidecar.istio.io/inject: 'false'
```

Comment:
Should be able to remove this I think? Big Bang had this in the past because of the job sidecar termination issue if I remember right. We have a pepr capability that should handle that so this should be able to be injected?

Reply:
Could also remove the `capabilities.drop` section too since that gets mutated in by Pepr 🤷

Reply:
Done 👍